code/game/gamemodes/clock_cult/clock_structures/eminence_spire.dm

```diff
@@ -35,6 +35,13 @@
 	else
 		nomination(user)
 
+/obj/structure/destructible/clockwork/eminence_spire/attack_drone(mob/living/simple_animal/drone/user)
+	if(!is_servantof_ratvar(user))
+		..()
+	else
+		to_chat(user, "<span class='notify'>You feel the omniscient gaze turn into a puzzled frown. Perhaps you should just stick to building.</span>")
```

Comment:
I'd use `warning` here. I don't think `notify` exists, maybe you're thinking of `notice`?